pkg/util/etcdutil/migrate.go

```diff
@@ -18,6 +19,8 @@ import (
 const (
 	apiserverAddr = "http://127.0.0.1:8080"
 	etcdServiceIP = "10.3.0.15"
+
+	etcdClusterName = "etcd-cluster"
```

Comment:
@aaronlevy we probably need a better name for it. self-hosted-etcd? system-etcd? kube-etcd?